hack/release/stable-pr.sh

```diff
@@ -1,42 +1,25 @@
 #!/usr/bin/env bash
 set -euo pipefail
 
-# updateKarpenterCoreGoMod bumps the karpenter-core go.mod to the release version so that the
-# karpenter and karpenter-core release versions match
-updateKarpenterCoreGoMod(){
-  RELEASE_VERSION=$1
-  if [[ $GITHUB_ACCOUNT != $MAIN_GITHUB_ACCOUNT ]]; then
-    echo "not updating go mod for a repo other than the main repo"
-    return
-  fi
-  go get -u "sigs.k8s.io/karpenter@${RELEASE_VERSION}"
-  cd test
-  go get -u "sigs.k8s.io/karpenter@${RELEASE_VERSION}"
-  cd ..
-  make tidy
-}
-
-SCRIPT_DIR=$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &>/dev/null && pwd)
+SCRIPT_DIR="$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &>/dev/null && pwd)"
+# shellcheck source=hack/release/common.sh
 source "${SCRIPT_DIR}/common.sh"
 
-config
-
-GIT_TAG=$(git describe --exact-match --tags || echo "none")
-if [[ $(releaseType $GIT_TAG) != $RELEASE_TYPE_STABLE ]]; then
+git_tag="$(git describe --exact-match --tags || echo "none")"
+if [[ "${git_tag}" != v* ]]; then
   echo "Not a stable release. Missing required git tag."
   exit 1
 fi
 
-versionData "$GIT_TAG"
-updateKarpenterCoreGoMod "$GIT_TAG"
+updateKarpenterCoreGoMod "${git_tag}"
```

Comment:
I think we should be able to drop this command now

Reply:
Will do.